### PR TITLE
Minor gramar fixes to the top level README

### PR DESCRIPTION
**Pull request by tcharding**
A few trivial fixups for the README. The first three are typos the last one is minor punctuation fix.

Thanks,
Tobin.

### CODE REVIEW

File: secp256kfun/README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ secp256kfun = "0.2"
 
 ### Should use?
 
-This library is ready for production as long what you are trying to produce is **fun and amusement!**.
+This library is ready for production as long what you are trying to produce is for **fun and amusement!**.
```
Comment (LLFourn) [inline secp256kfun/README.md:38]:
```suggestion
This library is ready for production as long what you are trying to produce is **fun and amusement!**.
```
"is fun and amusement" is correct here.

**Review by LLFourn — CHANGES_REQUESTED** [review]
Thanks very much for this ❤️ . One change I don't agree with.

File: secp256kfun/README.md (hunk 1 above)
Comment (tcharding) [reply secp256kfun/README.md:38]: Is this a meme I missed? I won't be offended if it is, I miss a lot of memes and internet jokes :)

Comment (tcharding) [reply secp256kfun/README.md:38]: oh now I read it, my bad.

**Review by LLFourn — APPROVED** [review]
Thanks!

**Pull request merged**